playbooks/production/roles/webserver/vars/main.yml

```diff
@@ -0,0 +1,7 @@
+vhosts:
+  - vhost_name: docs.kernelci.org
+    data_dir: /data/kernelci.org
```

Comment:
Would it make sense to align docs data dir to match the full vhost name like others, i.e. `/data/docs.kernelci.org`?

Reply:
Probably yes, i was just mapping things as they are now on legacy, i might add rename in recipe.

Reply:
Created https://github.com/kernelci/kernelci-deploy/issues/193 to track it post-merge